engine/src/flutter/shell/platform/darwin/macos/framework/Source/AccessibilityBridgeMacTest.mm

```diff
@@ -99,8 +99,9 @@ @implementation AccessibilityBridgeTestViewController
   auto bridge = std::static_pointer_cast<AccessibilityBridgeMacSpy>(
       viewController.accessibilityBridge.lock());
   FlutterSemanticsNode2 root;
+  FlutterSemanticsFlags flags;
   root.id = 0;
-  root.flags = static_cast<FlutterSemanticsFlag>(0);
+  root.flags2 = &flags;
```

Comment:
nit: the `flags` variable seems superfluous if its only used to pass an empty set of flags to `root.flags2`.

Reply:
Yes, but i want to pass a pointer, so I'm defining the flags first. I don't know if there's a better way to do it.